cypress/integration/smoke/dashboard.spec.js

```diff
@@ -0,0 +1,49 @@
+const SAFE = 'rin:0xFfDC1BcdeC18b1196e7FA04246295DE3A17972Ac'
+
+describe('Dashboard', () => {
+  before(() => {
+    // Go to the test Safe home page
+    cy.visit(`/${SAFE}/home`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  it('should display the dashboard title', () => {
+    cy.contains('main h1', 'Dashboard')
+  }) 
+
+  it('should display the overview widget', () => {
+    cy.contains('main p', SAFE).should('exist')
+    cy.contains('main', '1/9')
+    cy.get(`main a[href="/app/${SAFE}/balances"] button`).contains('View Assets')
+    cy.get(`main a[href="/app/${SAFE}/balances"]`).contains('Tokens3')
```

Comment:
If someone were to send another token to this safe the test would break. Should we remove the number or generalize it with regex?

Reply:
Not a fan of this approach. IMHO our test Safes should be picked specifically for these tests and never interacted with again.